models/utils.py

```diff
@@ -0,0 +1,43 @@
+import operator
+from datetime import datetime
+from dmutils.formats import DATE_FORMAT, DATETIME_FORMAT
+
+
+format_datetime_string_as_date = lambda x: datetime.strptime(x, DATETIME_FORMAT).strftime(DATE_FORMAT)
+remove_username_from_email_address = lambda x: '@{}'.format(x.split('@').pop())
+
+
+def process_collection(rules, collection):
+    if not rules:
+        return collection
+
+    for c in collection:
+        for k, v in rules.items():
+            if c.get(k):
+                c[k] = v(c[k])
+
+
+def return_filtered_collection(rules, collection):
+    if not rules:
+        return collection
+
+    # `contains` would be trickier to include because it switches the argument order
+    ops = {
+        '>': operator.gt,
+        '<': operator.lt,
+        '>=': operator.ge,
+        '<=': operator.le,
+        '==': operator.eq
+    }
```

Comment:
Yeah, I think this is stretching the bounds of what we want to be doing.

Works for our current use case though.